src/KeyDB.php

```diff
@@ -68,10 +61,12 @@ public static function getSupportedKeyTypes(): array
 
     /**
      * Constructs a new KeyDB.
+     *
+     * @param array $keys
+     *   Keys indexed by key ID.
      */
-    public function __construct()
+    public function __construct(protected array $keys = [])
```

Comment:
Why do we want this to be setable via constructor now?

Please on comment on either other changes link this and then re-assign to me.

Reply:
Good point. Must have missed that. Reverted (and removed the constructor entirely).